fidesops.toml

```diff
@@ -25,5 +25,5 @@ DRP_JWT_SECRET="secret"
 TASK_RETRY_COUNT=0
 TASK_RETRY_DELAY=1
 TASK_RETRY_BACKOFF=1
-REQUIRE_MANUAL_REQUEST_APPROVAL=false
+REQUIRE_MANUAL_REQUEST_APPROVAL=true
```

Comment:
Can we change this back to `false`? Right now the spec. is for Fidesops to default to automatically approving privacy requests.